From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Wed, 13 Oct 2010 13:10:49 -0700
Subject: staging: speakup: fix printk format warning

commit 0a652b96287ef61c97a2acab2bcc3d0f319b50e4 upstream.

Fix printk format warning:

drivers/staging/speakup/serialio.c:44: warning: format '%x' expects type 'unsigned int', but argument 2 has type 'long unsigned int'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: <speakup@braille.uwo.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/speakup/serialio.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/speakup/serialio.c b/drivers/staging/speakup/serialio.c
index bafd62f..65772ec 100644
--- a/drivers/staging/speakup/serialio.c
+++ b/drivers/staging/speakup/serialio.c
@@ -41,7 +41,7 @@ struct serial_state *spk_serial_init(int index)
 		__release_region(&ioport_resource, ser->port, 8);
 		err = synth_request_region(ser->port, 8);
 		if (err) {
-			pr_warn("Unable to allocate port at %x, errno %i", ser->port, err);
+			pr_warn("Unable to allocate port at %lx, errno %i", ser->port, err);
 			return NULL;
 		}
 	}
